### PR TITLE
[DOCS] Updating Tabs' Styles

### PR DESCRIPTION
**Pull request by deborahniesz**
# Description

Updated Tab styles to match mockup. Added a separated stylesheet to organize all tab changes (current styles and new ones).

The new styles are

 - Gray underline in Tabs group
 - Top borders are now rounded in the underline of a selected Tab .

[Jira Ticket](https://greatexpectations.atlassian.net/jira/software/c/projects/DSB/boards/79?issueParent=26175&selectedIssue=DSB-664)

[Netlify Preview](https://deploy-preview-9287.docs.greatexpectations.io/docs/home/)

https://github.com/great-expectations/great_expectations/assets/38264052/9257f921-a5e6-43a5-80ed-2d2fecc5656a

- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [ ] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted - run `invoke lint` (uses `black` + `ruff`)
- [ ] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

**Review by kwcanuck — APPROVED** [review]
I'm not a fan of the grey horizontal bar that displays when a tab isn't selected. I think this adds visual clutter and makes pages harder to scan quickly. I'll defer to the majority decision and the design experts. 

image

File: docs/docusaurus/src/css/tabs.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.tabs-container .margin-vert--md{
+  margin-top: 0 !important;
```
Comment (josectobar) [inline docs/docusaurus/src/css/tabs.scss]: Do we still need this?

File: docs/docusaurus/src/css/tabs.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+div[role="tabpanel"]{
+  --ifm-code-background: var(--ifm-alert-background-color-highlight);
+  border-radius: 0;
+  box-shadow:none;
+  border-top: 0;
+  padding: 0;
+}
```
Comment (josectobar) [inline docs/docusaurus/src/css/tabs.scss]: Same with this one, I noticed you moved these two but I can't tell where they're being applied

**Review by josectobar — CHANGES_REQUESTED** [review]
Left a small question.

**Review by josectobar — APPROVED** [review]
Looks great!

**Pull request merged**